transmitter_stub_test.js

```diff
@@ -0,0 +1,17 @@
+const {getTransmissionFailureCount, transmitInCelcius} = require("./transmitter")
+const {setIsActive, getRecievedCelcius}  = require("./stub")
+const {expect} = require("chai")
+
+setIsActive(true)
+transmitInCelcius(400.5);
+expect(getRecievedCelcius()).equals(204.72222222222223)
```

Comment:
Floating point cannot be checked for equality in a reliable way. Lookup the reason, then change this statement if you are convinced